pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -258,15 +258,6 @@ func (is *InstanceService) InstanceCreate(clusterName string, name string, confi
 	if config == nil {
 		return nil, fmt.Errorf("create Options need be specified to create instace")
 	}
-	if config.Trunk == true {
```

Comment:
Why is this commit removing this code? /me kind of confused

Reply:
must be another rebase bug. Sorry